manifests/sentinel.pp

```diff
@@ -129,13 +157,22 @@
   Integer[0] $parallel_sync = 1,
   Stdlib::Absolutepath $pid_file = $redis::params::sentinel_pid_file,
   Integer[1] $quorum = 2,
+  Optional[Enum['yes', 'no']] $sentinel_announce_hostnames = undef,
   Variant[Undef, Stdlib::IP::Address, Array[Stdlib::IP::Address]] $sentinel_bind = undef,
   Stdlib::Port $sentinel_port = 26379,
+  Optional[Enum['yes', 'no']] $sentinel_resolve_hostnames = undef,
+  Optional[Stdlib::Port] $sentinel_tls_port = undef,
```

Comment:
should we use https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/types/port/unprivileged.pp ? I guess noone wants to run this as root on a lowlevel port?

Reply:
I really doubt anyone would (or should).  I'm good with that change.  Again, mostly trying to be consistent with existing code with many of my choices.  I would do things very differently in this module myself (including using an epp).

Reply:
We use EPPs instead of ERBs as well. If you have some spare time to work on the conversion I'll be more than happy to review it :smile: